web_frontend/cloudscheduler/csv2/templates/csv2/status.html

```diff
@@ -408,16 +453,15 @@
 
                             <th class="col-spacer">&nbsp;</th>
 
-                            <th class="total-style" colspan="4">
```

Comment:
Seems like more volume stuff was removed here, I don't remember if we had a reason for doing so.